app/controllers/admin/events_controller.rb

```diff
@@ -29,6 +29,7 @@ def create
   end
 
   def update
+    @event = Event.find(params[:id])
     if @event.update(event_params)
       redirect_to edit_admin_event_path(@event), notice: alert_update(Event)
     else
```

Comment:
render action: :edit. Prefer to send error status such as 422 Unprocessable entity.